src/SUMMARY.md

```diff
@@ -118,6 +118,7 @@
 * [Data retention](security/data-retention.md)
 * [Encryption](security/encryption.md)
 * [Protective block](security/protective-block.md)
+* [Vulnerability Scanning and Penetration Testing](security/pen-test.md)
```

Comment:
We should probably use a slightly shorter title here, as the sidebar is not wide and users won't be able to read the whole thing anyway. :smile: I suggest "Security scans" or similar.

Reply:
Pushed revised version